bigcommerce/checkout-sdk-js
# feat(payment): PAYPAL-4067 added paypal fastlane shipping selector implementation to paypal commerce fastlane shipping strategy

## What?
Added paypal fastlane shipping selector implementation to paypal commerce fastlane shipping strategy

## Why?
To be able to use PayPal Fastlane shipping selector on the UI, so the customer will be able to select between addresses from Fastlane address book.

## Testing / Proof
Unit tests
Manual tests
CI

How it works:

https://github.com/bigcommerce/checkout-sdk-js/assets/25133454/cf646690-0022-40eb-979c-10f7bb380445

The flow should not work for customers who canceled authentication:

https://github.com/bigcommerce/checkout-sdk-js/assets/25133454/af03bc1b-d028-40c7-ae99-457191ecb8af
